### PR TITLE
Gen3: Update OV instructions

### PR DESCRIPTION
**Pull request by denysoblohin-okta**
## Description:

Follow-up for https://github.com/okta/okta-signin-widget/pull/3362

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [x] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [x] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-651065](https://oktainc.atlassian.net/browse/OKTA-651065)

### Screenshot/Video:

New page to enroll OV on same device ([mock](https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/data/idp/idx/authenticator-enroll-ov-same-device.json)):
authenticator-enroll-ov-same-device
Gen2:
authenticator-enroll-ov-same-device (g2)

New page to enroll OV on another device ([mock](https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/data/idp/idx/authenticator-enroll-ov-device-bootstrap.json)):
authenticator-enroll-ov-device-bootstrap
authenticator-enroll-ov-device-bootstrap (g2)

### CODE REVIEW

File: test/testcafe/framework/page-objects/EnrollOktaVerifyPageObject.js [hunk 1]
```diff
@@ -165,22 +165,37 @@ export default class EnrollOktaVerifyPageObject extends BasePageObject {
...
   getCopyOrgLinkButtonLabel() {
+    if (userVariables.v3) {
+      return this.form.getButton('Copy sign-in URL to clipboard').innerText;
+    }
     return this.getTextContent(COPY_ORG_LINK_BUTTON_CLASS);
   }
 
   getCopiedOrgLinkValue() {
+    if (userVariables.v3) {
+      return this.form.getElement('[data-se=o-form-explain] > .no-translate').innerText;
```
[the remark this answers is not in the document]
Comment (denysoblohin-okta) [reply test/testcafe/framework/page-objects/EnrollOktaVerifyPageObject.js]: Ok, changed to skip the assertion

File: playground/mocks/data/idp/idx/authenticator-enroll-ov-device-bootstrap-multiple-devices.json [hunk 2]
```diff
@@ -176,7 +176,7 @@
                     ]
                   }
                 },
-                "relatesTo": "$.authenticators.value[3]"
+                "relatesTo": "$.authenticators.value[2]"
```
Comment (shawyu-okta) [inline playground/mocks/data/idp/idx/authenticator-enroll-ov-device-bootstrap-multiple-devices.json:179]: Were these `relatesTo` values in the mocks incorrect?

**Review by shawyu-okta — APPROVED** [review]
Can you add unit test coverage for the transformer? There are some new branches to cover so let's make sure we test them.

Otherwise lgtm

File: playground/mocks/data/idp/idx/authenticator-enroll-ov-device-bootstrap-multiple-devices.json (hunk 2 above)
Comment (denysoblohin-okta) [reply playground/mocks/data/idp/idx/authenticator-enroll-ov-device-bootstrap-multiple-devices.json:179]: Yes. This causes `authClient.idx.proceed/start` to throw exception

**Pull request merged**